libdnf/dnf-transaction.cpp

```diff
@@ -1481,14 +1485,8 @@ dnf_transaction_commit(DnfTransaction *transaction, HyGoal goal, DnfState *state
     if (!ret)
         goto out;
 
-    // finalize swdb transaction
-    // always load a new sack with rpmdb state after the transaction
-    rpmdb_version_sack = dnf_sack_new();
-    dnf_sack_load_system_repo(rpmdb_version_sack, nullptr, DNF_SACK_LOAD_FLAG_NONE, nullptr);
-    rpmdb_end = dnf_sack_get_rpmdb_version(rpmdb_version_sack);
-    g_object_unref(rpmdb_version_sack);
-
-    swdb->endTransaction(_get_current_time(), rpmdb_end.c_str(), libdnf::TransactionState::DONE);
+    rpmdb_end.reset(rpmdbCookie(rpmtsGetRdb(priv->ts)));
```

Comment:
Please can you also report when `rpmdbCookie(rpmtsGetRdb(priv->ts))` returns `NULL`?

Reply:
OK.